### ISSUE TITLE
'Share with QR' crashes the app

### ISSUE THREAD

**Issue by enihcam** [shadowsocks/shadowsocks-android#2348]
Please read contributing guidelines. Thanks.

**Describe the bug**
'Share with QR' crashes the app

**To Reproduce**
1. Go to 'shadowsocks'
2. Click on 'Share' icon
3. Choose 'QR code'
4. App exits

**Expected behavior**
QR code showing up

**Screenshots**

**Smartphone (please complete the following information):**
 - Android/Chrome OS version: 9 PKQ1.181007.001
 - Device: Mi Note 3
 - Version: 4.8.5
 - Last version that did not exhibit the issue: before update from MIUI 10 to MIUI 11

**Configuration**
Put an `x` inside the [ ] that applies.

* [x] IPv4 server address
* [ ] IPv6 server address
* [x] Client IPv4 availability
* [ ] Client IPv6 availability
* Encrypt method:
* Route
  * [x] All
  * [ ] Bypass LAN
  * [ ] Bypass China
  * [ ] Bypass LAN & China
  * [ ] GFW List
  * [ ] China List
  * [ ] Custom rules
* [ ] IPv6 route
* [ ] Apps VPN mode
  * [ ] Bypass mode
* Remote DNS: 8.8.8.8
* [x] DNS over UDP
* Plugin configuration (if applicable):
* [ ] Auto Connect
* [x] TCP Fast Open
* If you're not using VPN mode, please supply more details here:

**Additional context**

**Comment by Mygod** [comment]
Impossible to investigate. Please provide logcat.

**Comment by Mygod** [comment]
Okay seems like it's panic caused by `com.google.zxing.WriterException: Data too big`.

**Comment by shadowsocksRb** [comment]
建议更新qrgen到2.6.0，对方已经更换名字和仓库。
```
repositories {
    maven { url "https://jitpack.io" }
}
dependencies {
    implementation 'com.github.kenglxn.QRGen:android:2.6.0'
}
```

**Comment by Mygod** [comment]
@shadowsocksRb Addressed via 09e2948d.
